src/main/resources/db.changeLog/changeLog-master.xml

```diff
@@ -2,7 +2,35 @@
 <databaseChangeLog
         xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:pro="http://www.liquibase.org/xml/ns/pro"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
-    <includeAll path="com/example/changelog/"/>
-</databaseChangeLog>
\ No newline at end of file
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+		http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+		http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+    <preConditions>
+        <runningAs  username="bestuser"/>
+    </preConditions>
+    <changeSet id ="1" author="bestuser">
```

Comment:
`changeLog-master.xml` file must include paths to `changeLog-1.xml`, `changeLog-2.xml` and others `changeLog-X.xml` you will create in future.

```
<databaseChangeLog ...>
    <include file="db.changeLog/changeLog-1.xml" relativeToChangelogFile="false"/>
    <include file="db.changeLog/changeLog-2.xml" relativeToChangelogFile="false"/>
    <include file="db.changeLog/changeLog-3.xml" relativeToChangelogFile="false"/>
    <include file="db.changeLog/changeLog-X.xml" relativeToChangelogFile="false"/>
</databaseChangeLog>
```